<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewExchangePopup</class>
 <widget class="QDialog" name="NewExchangePopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Exchange</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QPushButton" name="folder_button">
       <property name="text">
        <string>Select Folder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Source: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Exchange ID: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="exchange_id_edit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="folder_path_label"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="freq_combo">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Day1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hour4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hour1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Min30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Min15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Min5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Min1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Second1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tick</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Frequency:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="submit_button">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
